controllers/metal3.io/baremetalhost_controller.go

```diff
@@ -446,6 +446,18 @@ func inspectionDisabled(host *metal3v1alpha1.BareMetalHost) bool {
 	return false
 }
 
+// hasInspectAnnotation checks for existence of inspect.metal3.io annotation
+// and returns true if it exist
+func hasInspectAnnotation(host *metal3v1alpha1.BareMetalHost) bool {
+	annotations := host.GetAnnotations()
+	if annotations != nil {
+		if _, ok := annotations[inspectAnnotationPrefix]; ok && annotations[inspectAnnotationPrefix] != "disabled" {
```

Comment:
nit: this is a slightly repetitive way of writing:

```
if expect, ok := annotations[inspectAnnotationPrefix]; ok && expect != "disabled" {
```

Reply:
fixed, thanks